$DragonFly: src/gnu/usr.bin/cvs/lib/Attic/save-cwd.c.patch,v 1.1 2005/12/03 12:11:44 corecode Exp $
Index: save-cwd.c
===================================================================
RCS file: /space/cvs/dragonfly/src/contrib/cvs-1.12.12/lib/save-cwd.c,v
retrieving revision 1.1
diff -u -r1.1 save-cwd.c
--- save-cwd.c	21 Apr 2005 06:33:15 -0000	1.1
+++ save-cwd.c	16 Oct 2005 10:16:48 -0000
@@ -41,7 +41,6 @@
 
 #include <errno.h>
 
-#include "chdir-long.h"
 #include "xgetcwd.h"
 
 /* On systems without the fchdir function (WOE), pretend that open
@@ -105,7 +104,7 @@
   if (0 <= cwd->desc)
     return fchdir (cwd->desc);
   else
-    return chdir_long (cwd->name);
+    return -1;
 }
 
 void
